telco-examples/mgmt-cluster/airgap/eib/mgmt-cluster-airgap.yaml

```diff
@@ -85,31 +84,59 @@ kubernetes:
       - name: rancher
         url: https://charts.rancher.io
       - name: suse-edge-charts
-        url: oci://registry.suse.com/edge/3.2
+        url: oci://registry.suse.com/edge/charts
       - name: rancher-prime
         url: https://charts.rancher.com/server-charts/prime
 embeddedArtifactRegistry:
   images:
-    - name: registry.suse.com/rancher/cluster-api-controller:v1.8.4
-    - name: registry.suse.com/rancher/cluster-api-provider-rke2-bootstrap:v0.9.0
-    - name: registry.suse.com/rancher/cluster-api-provider-rke2-controlplane:v0.9.0
-    - name: registry.rancher.com/rancher/cluster-api-metal3-controller:v1.8.2
-    - name: registry.rancher.com/rancher/cluster-api-metal3-ipam-controller:v1.8.1
-    - name: registry.rancher.com/rancher/cluster-api-operator:v0.14.0
-    - name: registry.rancher.com/rancher/kubectl:v1.29.2
-    - name: registry.rancher.com/rancher/fleet-agent:v0.11.2
-    - name: registry.rancher.com/rancher/fleet:v0.11.2
-    - name: registry.rancher.com/rancher/rancher-webhook:v0.6.2
-    - name: registry.rancher.com/rancher/mirrored-longhornio-csi-attacher:v4.7.0
-    - name: registry.rancher.com/rancher/mirrored-longhornio-csi-node-driver-registrar:v2.12.0
-    - name: registry.rancher.com/rancher/mirrored-longhornio-csi-provisioner:v4.0.1-20241007
-    - name: registry.rancher.com/rancher/mirrored-longhornio-csi-resizer:v1.12.0
-    - name: registry.rancher.com/rancher/mirrored-longhornio-csi-snapshotter:v7.0.2-20241007
-    - name: registry.rancher.com/rancher/mirrored-longhornio-livenessprobe:v2.14.0
-    - name: registry.rancher.com/rancher/mirrored-longhornio-longhorn-engine:v1.7.2
-    - name: registry.rancher.com/rancher/mirrored-longhornio-longhorn-instance-manager:v1.7.2
-    - name: registry.rancher.com/rancher/mirrored-longhornio-longhorn-manager:v1.7.2
-    - name: registry.rancher.com/rancher/mirrored-longhornio-longhorn-share-manager:v1.7.2
-    - name: registry.rancher.com/rancher/mirrored-longhornio-longhorn-ui:v1.7.2
-    - name: registry.rancher.com/rancher/system-upgrade-controller:v0.14.2
-    - name: registry.suse.com/rancher/seedimage-builder:1.6.4
+    - name: quay.io/jetstack/cert-manager-cainjector:v1.14.2
+    - name: quay.io/jetstack/cert-manager-controller:v1.14.2
+    - name: quay.io/jetstack/cert-manager-webhook:v1.14.2
```

Comment:
The cert-manager version specified above is `1.15.3` 🤔 Perhaps this can be removed and EIB can derive the images from the chart - I'm unclear why the versions don't match though

Reply:
push forced with the minimum list to be tested in airgap environment